qiita_core/configuration_manager.py

```diff
@@ -107,6 +107,8 @@ class ConfigurationManager(object):
         The filepath for the configuration file that is loaded
     portal_fp : str
         The filepath to the portal styling config file
+    plugin_launcher : str
```

Comment:
Should we have some user-level documentation on what this script looks like, maybe in the Admin Tutorials section? Alternatively if we don't expect this to change or be modified, then we probably don't need that.